src/main.cpp

```diff
@@ -35,14 +34,20 @@
 
 using namespace LAMMPS_NS;
 
+// for convenience
+static void finalize()
```

Comment:
Couldn't we make that part of the library interface?

Reply:
It would feel more natural to me if there would be a function that finalizes all the things that lammps initializes automatically.

Reply:
These functions are primarily available for internal use so this is not much of a concern. Those have grown organically. As mentioned elsewhere in this PR, I don't want to rock the boat too much at the current state of development when we are slowly converging toward a stable release. These kinds of changes are better done at the beginning of a development cycle.